bindings/python/tasks/velocity.hpp

```diff
@@ -70,9 +70,13 @@ void pyVelocityCartesianAdmittance(py::module & m) {
     py::class_<CartesianAdmittance, std::shared_ptr<CartesianAdmittance>, Cartesian>(m, "CartesianAdmittance")
         .def(py::init<std::string, XBot::ModelInterface&, std::string, XBot::ForceTorqueSensor::ConstPtr>())
         .def("getCartesianCompliance",py::overload_cast<>(&CartesianAdmittance::getCartesianCompliance))
+        .def("getFilterOmega",py::overload_cast<>(&CartesianAdmittance::getFilterOmega))
+        .def("getFilterTimeStep",py::overload_cast<>(&CartesianAdmittance::getFilterTimeStep))
         .def("setWrenchReference", py::overload_cast<const Eigen::Vector6d&>(&CartesianAdmittance::setWrenchReference))
         .def("getWrenchReference", py::overload_cast<>(&CartesianAdmittance::getWrenchReference))
+        .def("getLambda", py::overload_cast<>(&CartesianAdmittance::getLambda))
         .def("setFilterDamping", &CartesianAdmittance::setFilterDamping)
+        .def("setFilterTimeStep", &CartesianAdmittance::setFilterTimeStep)
```

Comment:
```suggestion
        .def("getFilterOmega", &CartesianAdmittance::getFilterOmega)
        .def("getFilterTimeStep", &CartesianAdmittance::getFilterTimeStep)
        .def("setWrenchReference", py::overload_cast<const Eigen::Vector6d&>(&CartesianAdmittance::setWrenchReference))
        .def("getWrenchReference", py::overload_cast<>(&CartesianAdmittance::getWrenchReference))
        .def("getLambda", &CartesianAdmittance::getLambda)
        .def("setFilterDamping", &CartesianAdmittance::setFilterDamping)
        .def("setFilterTimeStep", &CartesianAdmittance::setFilterTimeStep)
```
I don't think these casts are necessary, since there is only one version of these functions.